CharlieBackend.AdminPanel/Utils/HttpUtil.cs

```diff
@@ -19,15 +19,15 @@ public HttpUtil()
             _client = new HttpClient();
```

Comment:
Our project configured to send HTTPs data. Willnt it affect on https data type?

Reply:
No, this doesn't affect our project in any way.